<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="pskindero-ThinkPad-Edge-E545" name="com.pskindero.javaee.javaeetech.jpa.model.copy.EmployeeDaoImplTest" tests="2" failures="0" timestamp="22 May 2017 16:49:33 GMT" time="0.635" errors="1">
  <testcase name="setup" time="0.635" classname="com.pskindero.javaee.javaeetech.jpa.model.copy.EmployeeDaoImplTest">
    <error type="javax.persistence.PersistenceException" message="No Persistence provider for EntityManager named employee_db">
      <![CDATA[javax.persistence.PersistenceException: No Persistence provider for EntityManager named employee_db
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:61)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:39)
	at com.pskindero.javaee.javaeetech.jpa.dao.GenericDaoImpl.<init>(GenericDaoImpl.java:23)
	at com.pskindero.javaee.javaeetech.jpa.dao.impl.EmployeesDaoImpl.<init>(EmployeesDaoImpl.java:11)
	at com.pskindero.javaee.javaeetech.jpa.model.copy.EmployeeDaoImplTest.setup(EmployeeDaoImplTest.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase name="readExistingEmployee" time="0.000" classname="com.pskindero.javaee.javaeetech.jpa.model.copy.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- readExistingEmployee -->
</testsuite> <!-- com.pskindero.javaee.javaeetech.jpa.model.copy.EmployeeDaoImplTest -->
